src/tuplet.ts

```diff
@@ -150,15 +140,15 @@ export class Tuplet extends Element {
    * Set whether or not the bracket is drawn.
    */
   setBracketed(bracketed: boolean): this {
-    this.bracketed = !!bracketed;
+    this.options.bracketed = !!bracketed;
```

Comment:
Do we still need to use the not not here to force the param to be boolean? I think it was a vestige of classic JS (see VexFlow 3.0.9 at https://github.com/0xfe/vexflow/blob/00ec15c67ff333ea49f4d3defbd9e22374c03684/src/tuplet.js#L123).